Doc/library/http.client.rst

```diff
@@ -460,6 +460,14 @@ statement.
 
    HTTP protocol version used by server.  10 for HTTP/1.0, 11 for HTTP/1.1.
 
+.. attribute:: HTTPResponse.url
+
+   URL of the resource retrieved, commonly used to determine if a redirect was followed
+
+.. attribute:: HTTPResponse.headers
+
+   Headers of the response in the form of an :meth:`email.message_from_string` instance
```

Comment:
message_from_string is a function, not a class.  Better doc suggestion: `in the form of an EmailMessage instance`